MHRA/products
# API Pipeline secrets 🕵️‍♂️

# API Pipeline secrets 🕵️‍♂️

[Airtable ticket](https://airtable.com/tbl7CISHKkr8Kdeh2/viwlDIU6cp8cbQYhj/recbVwgdO7MEOYziQ?blocks=hide)
GitHub Issue #376 

![](https://media.giphy.com/media/MWfjcQsYkfzBS/giphy.gif)

### Acceptance Criteria

- [ ] API POD to consume environment variables kubernetes secret

### Testing information

1. Go to `infrastructure/environments/development`folder
```sh
infrastructure/environments/development
```
2. Source environmental variables
```sh
source .env
```
3. Refresh cluster credentials
```sh
../../scripts/create-kubernetes-config.sh
```
4. Select option **`3) dev`**
5. See **`azure-search-secrets`** in the cluster
```sh
kubectl describe secrets/azure-search-secrets
```
6. 🍾
